Utilities/src/test/java/edu/stanford/hivdb/mutations/CodonTranslationTest.java

```diff
@@ -37,10 +37,18 @@ public void testConstructor() throws Exception {
 
 	@Test
 	public void testGenerateControlString() {
-
 		assertEquals(
 			"  .", CodonTranslation.generateControlString("TTT", "Asn"));
-
+		
+		assertEquals(
+			"  .", CodonTranslation.generateControlString("TTTT", "Asn"));
```

Comment:
Actually an exception is more proper here for non-triplet nucleotides. Can you open an issue of this? Just record it, we don't have to fix it right now.